### PR TITLE
Add help.1 file

### PR DESCRIPTION
**Pull request by hhorak**
Atomic CLI tool shows groff formatted man page if called by atomic help , so all images should support it.
Currently the man page is generated outside of the build system by pandoc -s -f markdown -t man.

### CODE REVIEW

File: hack/run_test [hunk 1]
```diff
@@ -498,6 +498,31 @@ $volume_options
...
+  echo "  Testing documentation in the container image"
+  # Extract the help files from the container
+  for f in help.md help.1 ; do
+    docker run --rm ${IMAGE_NAME} /bin/bash -c "cat /${f}" >${tmpdir}/${f}
+  done
+  # Check whether the files use the correct format
+  if ! file ${tmpdir}/help.1 | grep -q roff ; then
+    echo "ERROR: /help.1 is not in troff or groff format"
+    return 1
+  fi
```
Comment (praiskup) [inline hack/run_test]: At least this first part could be in `container-common-scripts`.

File: hack/run_test [hunk 2]
```diff
@@ -506,6 +531,7 @@ function run_general_tests() {
   DOCKER_ARGS="-u 12345" PGUSER=user2 PASS=pass run_tests no_admin_altuid
   DOCKER_ARGS="-u 12345" PGUSER=user3 PASS=pass1 ADMIN_PASS=r00t run_tests admin_altuid
   DB=postgres DOCKER_ARGS="-u 12345" ADMIN_PASS=rOOt run_tests only_admin_altuid
+  run_doc_check
```
Comment (praiskup) [inline hack/run_test]: In case this function stays in postgresql-container sources, can we have it in `TEST_LIST=`

**Comment by praiskup** [comment]
Not something we could change here in the PR, but placing documentation directly into '/' directory is a bit weird atomic decision :).

Looks good otherwise.

**Comment by praiskup** [comment]
Btw., rfe, we could have common.mk command to generate help.1 by pandoc ... (maintainer-only make target)

**Comment by hhorak** [comment]
> Btw., rfe, we could have common.mk command to generate help.1 by pandoc ... (maintainer-only make target)

Good idea!

File: hack/run_test (hunk 1 above)
Comment (hhorak) [inline hack/run_test]: Makes sense to me, as soon as we start detaching parts of the tests to common part, this should be one of the first suspects. I've include that in the issue so we don't forget:
https://github.com/sclorg/container-common-scripts/issues/1

File: hack/run_test (hunk 2 above)
Comment (hhorak) [inline hack/run_test]: Just created a new commit, that does this.

**Comment by hhorak** [comment]
[test]

**Comment by praiskup** [comment]
Without having a look how the man page is generated (it seems to be readable), the PR looks fine to me.  Thanks.

**Comment by pkubatrh** [comment]
Running the command `pandoc -s -f markdown -t man` manually on the README files results in some differences:

--- generated	2017-02-23 08:30:17.463061671 +0100
+++ root/help.1	2017-02-23 08:27:54.391512867 +0100
@@ -1,7 +1,10 @@
 .\"t
+.\" WARNING: Do not edit this file manually, it is generated from README.md automatically.
+.\"
+.\"t
 .\" Automatically generated by Pandoc 1.16.0.2
 .\"
-.TH "" "" "" "" ""
+.TH "POSTGRESQL\-95\-RHEL7" "1" "February 22, 2017" "Container Image Pages" ""
 .hy
 .SH PostgreSQL Docker image
 .PP
@@ -183,3 +186,5 @@
 between the values stored in the variables and the actual passwords.
 Whenever a database container starts it will reset the passwords to the
 values stored in the environment variables.
+.SH AUTHORS
+Red Hat.

Posting it here so that once we are generating this via a make target we should remember to add these parts to the help page.

**Comment by praiskup** [comment]
Thanks.  I think maintainers would have to review the changes manually while committing them to git anyway, so it is not huge issue.  Pandoc is somewhat unstable tool, from my previous experience and guess we can expect random differences.

**Pull request merged**